ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.11.yaml

```diff
@@ -76,6 +76,17 @@ resources:
       memory: 4Gi
 test_binary_build_commands: make test
 tests:
+- as: e2e-aws-ipi
```

Comment:
@liangxia 
1, please add test into ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.11__periodics.yaml, not openshift-openshift-tests-private-release-4.11.yaml
2, in that file, please replace the test
```yaml
- as: extended-aws-ipi
  cron: 0 2 */3 * *
  steps:
    cluster_profile: aws-qe
    env:
      BASE_DOMAIN: qe.devcluster.openshift.com
    test:
    - ref: openshift-extended-test
    workflow: cucushift-installer-rehearse-aws-ipi
```
to
```yaml
- as: e2e-aws-ipi
  cron: 0 2 * * *
  steps:
    cluster_profile: aws-qe
    env:
      BASE_DOMAIN: qe.devcluster.openshift.com
      E2E_RUN_TAGS: '@aws-ipi and @network-openshiftsdn and not @fips'
      TAG_VERSION: '@4.11'
    test:
    - chain: openshift-e2e-test-qe
    workflow: cucushift-installer-rehearse-aws-ipi
```

3, the different between `cron: 0 2 * * *` and `interval 24h` is that `interval 24h` possible cause the test trigger peak. so I prefer to `cron: 0 2 * * *` to set trigger time to distribute it.

Reply:
For 3, `interval 24h` is preferred IMO. The scheduler will take care of it when it will be executed, especially given we have dozens of profiles upcoming.

Reply:
if using `interval 24`, after we have dozens of profiles, it will almost start at same time per my observations to cause peak. so I prefer to control it so that they are distributed averagely.
you could see the test in 4.11__periodics.yaml and how we distribute them.

Reply:
Definitely, that's not true. 
You can search "verification-tests" in https://testgrid.k8s.io/redhat-openshift-ocp-release-4.11-informing, you can see those jobs are spread across the day.

Reply:
it is because you submit the tests with in different PR and merged at different time
here is the definition of interval `Interval is how frequently the test should be run based on the last time the test ran. Setting this field will create a periodic job instead of a presubmit`

so, if you submit more than one tests in the PR (for example 3 tests or 5 tests), these tests will start at almost same time to cause possible peak.
So, in order to avoid test trigger peak, `cron xxx` is better than `interval 24h` which you lost the control to average the test trigger.